pyperformance/data-files/requirements.txt

```diff
@@ -10,7 +10,7 @@ psutil==5.8.0
     # via -r requirements.in
 pyparsing==3.0.6
     # via packaging
-pyperf==2.3.0
+pyperf @ file:///home/cappadokes/code/pyperf
```

Comment:
This seems like something you changed locally, but that should be changed back before landing this.

Reply:
Unfortunately this is somewhat tricky:
- for `--track-energy` to work, `pyperf` should also support energy measurements. There is no such `pyperf` release at the moment, but [we are working on it](https://github.com/psf/pyperf/pull/125).
- until then, if a user wants to use `--track-energy` from `pyperformance`, she must indeed use a local `pyperf` in `requirements.txt`.